package.json

```
@@ -66,7 +65,7 @@
   },
   "lint-staged": {
     "*.ts": [
-      "yarn prettier --write",
+      "bun prettier --write",
```

```suggestion
      "prettier --write",
```

will use the installed package manager